daffodil-test/src/test/resources/org/apache/daffodil/section17/calc_value_properties/outputValueCalc2.tdml

```diff
@@ -494,7 +494,7 @@
       </tdml:dfdlInfoset>
     </tdml:infoset>
     <tdml:document>
-      <tdml:documentPart type="text">XYZ5.0X038.0Y41.23Z008</tdml:documentPart>
+      <tdml:documentPart type="text">XYZ5X038.0Y41.23Z008</tdml:documentPart>
```

Comment:
ICU change behabior here, the text pattern here has option decimal point for the first number (5.0e03). The pattern says the .0 is optional, and so is now correctly left off when the value is zero.